nipype/pipeline/plugins/tests/test_base.py

```diff
@@ -39,3 +40,34 @@ def func(arg1):
 
 wf.run(plugin='MultiProc')
 """
+
+@pytest.mark.regression
+def test_remove_nodes(tmpdir):
+    import os
+    import nipype.interfaces.utility as niu
+    import nipype.pipeline.engine as pe
+
+    wf = pe.Workflow(name='test')
+
+    def func(arg1):
+        try:
+            if arg1 == 2:
+                raise Exception('arg cannot be ' + str(arg1))
+        except:
+            pass
```

Comment:
Is the goal to fail or not? If not, we can skip the entire `try`/`except` block. If so, we should not catch the exception.